modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -14,8 +14,43 @@ include::partial$_show_page_header_block.adoc[]
 
 
 NOTE: Couchbase Lite 3.0 introduces some breaking changes. +
-If you're upgrading from 2.x, refer to the appropriate upgrade page -- see: <<lbl-upgrade>>
-Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
+If you're upgrading from 2.x, refer to the appropriate upgrade page -- see: <<lbl-upgrade>>. +
+You should be able to upgrade from 3.0.x to 3.1.x without manual intervention.
+
+== Release 3.2.3 (April 2025)
+
+=== New Features
+
+==== Reactive APIs in Swift
```

Comment:
This could be misleading, as we don’t provide any new reactive APIs in the way developers might expect from functional reactive programming (e.g., RxSwift). In my opinion, a more general term like "Swift API Enhancements" would be more appropriate.

Reply:
Updated heading